tests/core/test_graph.py

```diff
@@ -1970,13 +1970,13 @@ def test_node_degrees(use_ilocs):
     degrees = g.node_degrees(use_ilocs=use_ilocs)
 
     # expected node degrees - keys are node ids
-    expected = {0: 1, 1: 2, 2: 1, 3: 1, 4: 4, 5: 3}
+    expected = {0: 1, 1: 2, 2: 1, 3: 1, 4: 4, 5: 3, 6: 0}
     if use_ilocs:
         for node_id in expected.keys():
             node_iloc = g.node_ids_to_ilocs([node_id])[0]
             assert expected[node_id] == degrees[node_iloc]
     else:
-        assert expected == degrees
+        assert_items_equal(expected, degrees)
```

Comment:
This no longer _needs_ to change:

```suggestion
        assert expected == degrees
```